Tutorial doc fixes: tutorials 1 & 2

maindlt opened the pull request:
Updated tutorials `1_data_prep_for_gauss_hermites.ipynb` and `2_quickstart.ipynb`.
Biggest changes:
- `2_quickstart.ipynb` now describes the only supported way to create and run models: using the `ModelIterator` class.
- Tutorials don't have the output as part of the distributed notebooks
- The automatically created tutorial output in the docs does no longer have full paths (potentially containing personal information)

Some minor doc tweaks.

Please have a look at the tutorials 1 and (especially) 2...

Closes #287.

azocchi commented:
Hello @maindlt, I have tested the new versions of the tutorials, and I think that they are ok! However, please notice that the file `NGC6278_config_single.yaml` is not available in the tutorial directory -- I imagine that this is due to the fact that we are waiting for PR #263 to be merged before including this (to be able to use the new parameter generator), but I am writing this down so we don't forget! ;)
Anyway, I committed some changes, mainly fixing some typos, and I have changed the color maps used in the data preparation file, to make them coherent with the ones used in our `Plotter` class. Please notice that this will only work with the most recent version of `pafit` (2.0.8) -- can you please double check this and, in case this is needed, update the requirements accordingly? Thank you! :)

maindlt commented:
Many thanks for thoroughly going through the tutorials 1 and 2 and for your improvements, @azocchi :-)

The config file was indeed missing in my earlier commit (sorry...) - I added it in the last commit and yes, we need to change the parameter generator in it to `SpecificModels` (and perhaps include more specific exercises working with a few explicit parameter sets at the end) after merging PR #263 ;-)

Also, I double checked the dependency on the `pafit` version: `pafit 2.0.7` results in an error while the tutorial 1 works with `pafit 2.0.8`. I updated the requirements accordingly.

Cheers,
Thomas

prashjet commented:
Notebooks 1 and 2 run successfully from start to finish.

Happy to merge.
